source/frontend/src/components/common/mapFSM/machineDefinition/mapMachine.ts

```diff
@@ -366,13 +366,14 @@ const advancedFilterSideBarStates = {
         },
       },
     },
-    filtering: {
-      entry: [send({ type: 'REFRESH_PROPERTIES', searchCriteria: defaultPropertyFilter })],
-      exit: [send({ type: 'REFRESH_PROPERTIES' })],
+    mapFilterOpened: {
+      entry: [
+        send({ type: 'REFRESH_PROPERTIES', searchCriteria: defaultPropertyFilter }),
```

Comment:
Hmm, not sure if this makes sense anymore. Right now, every time I open the sidebar the property refresh event fires, even if I don't change anything within the advanced filter. Given that I haven't changed the filter, I'm not sure why a refresh would be necessary.